internal/pkg/service/stream/storage/level/local/diskwriter/network/config.go

```diff
@@ -42,5 +43,6 @@ func NewConfig() Config {
 		ShutdownTimeout:        30 * time.Second,
 		KCPInputBuffer:         8 * datasize.MB,
 		KCPResponseBuffer:      512 * datasize.KB,
+		PipelineBalancer:       "rand",
```

Comment:
Can you use the constant please? If there is an dependencies issues, move constant to the network package - where the configuration structure is.

Reply:
There is dependency issue, i'll move it